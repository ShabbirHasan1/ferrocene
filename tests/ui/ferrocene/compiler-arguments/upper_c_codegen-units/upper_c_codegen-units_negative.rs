// check-fail
// compile-flags: -Ccodegen-units=-1

fn main() {}

// ferrocene-annotations: um_rustc_C_codegen_units
