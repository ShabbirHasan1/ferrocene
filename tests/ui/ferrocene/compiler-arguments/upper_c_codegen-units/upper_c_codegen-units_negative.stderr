error: incorrect value `-1` for codegen option `codegen-units` - a number was expected

